apps/src/craft/designer/craft.js

```diff
@@ -0,0 +1,913 @@
+/* global trackEvent */
+import $ from 'jquery';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import _ from 'lodash';
+import Hammer from "hammerjs";
+
+var studioApp = require('../../StudioApp').singleton;
+var commonMsg = require('@cdo/locale');
+var craftMsg = require('./locale');
+var codegen = require('../../codegen');
+var GameController = require('./game/GameController');
+import FacingDirection from './game/LevelMVC/FacingDirection';
+var dom = require('../../dom');
+var eventsLevelbuilderOverrides = require('./eventsLevelbuilderOverrides');
+var MusicController = require('../../MusicController');
+var Provider = require('react-redux').Provider;
+var AppView = require('../../templates/AppView');
+var CraftVisualizationColumn = require('./CraftVisualizationColumn');
+import {entityActionBlocks, entityActionTargetDropdownBlocks} from './blocks';
+
+var TestResults = studioApp.TestResults;
+
+var MEDIA_URL = '/blockly/media/craft/';
+
+var ArrowIds = {
+  LEFT: 'leftButton',
+  UP: 'upButton',
+  RIGHT: 'rightButton',
+  DOWN: 'downButton'
+};
+
+/**
+ * Create a namespace for the application.
+ */
+var Craft = module.exports;
```

Comment:
Do we essentially have two different version/instances of Craft then? Is this one back compat/replacing the old one, or do we expect to run both in parallel?